testpilot/frontend/static-src/app/views/experiment-row-view.js

```diff
@@ -69,6 +69,7 @@ export default BaseView.extend({
     this.loggedIn = opts.loggedIn;
   },
 
+// These need to be changed to fit tab mapping: Just Launched, Just Updated, Enabled
```

Comment:
I think this comment could be removed from here, and added as a task to the followup bug to add just launched / just updated support.